src/System.Security.Cryptography.Csp/tests/RC2CryptoServiceProviderTests.cs

```diff
@@ -27,5 +30,22 @@ public static void RC2KeySize()
                 Assert.Throws<CryptographicException>(() => rc2.KeySize = 128 + 8);
             }
```

Comment:
I'll add the NoSalt tests (pos and neg) w.r.t. validation on setting to true, but I don't think it does anything on any later\supported versions of Windows